packages/@ama-sdk/core/README.md

```diff
@@ -1,40 +1,40 @@
 # @ama-sdk Core
 
-This package contains all the [plugins](/packages/@ama-sdk/core/src/plugins), helpers and object definitions to dialog with an API following the `ama-sdk` architecture.
+This package contains all the [plugins](src/plugins), helpers and object definitions to dialog with an API following the `ama-sdk` architecture.
 
-Please refer to the [generator-sdk](/packages/@ama-sdk/generator-sdk/README.md) package for getting started with an API based on `ama-sdk`.
+Please refer to the [generator-sdk](../generator-sdk/README.md) package for getting started with an API based on `ama-sdk`.
 
 ## Available plugins
 
-- [additional-params](/packages/@ama-sdk/core/src/plugins/additional-params)
-- [api-configuration-override](/packages/@ama-sdk/core/src/plugins/api-configuration-override)
-- [api-key](/packages/@ama-sdk/core/src/plugins/api-key)
-- [bot-protection-fingerprint](/packages/@ama-sdk/core/src/plugins/bot-protection-fingerprint)
-- [concurrent](/packages/@ama-sdk/core/src/plugins/concurrent)
-- [core](/packages/@ama-sdk/core/src/plugins/core)
-- [client-facts](/packages/@ama-sdk/core/src/plugins/client-facts)
-- [custom-info](/packages/@ama-sdk/core/src/plugins/custom-info)
-- [exception](/packages/@ama-sdk/core/src/plugins/exception)
-- [fetch-cache](/packages/@ama-sdk/core/src/plugins/fetch-cache)
-- [fetch-credentials](/packages/@ama-sdk/core/src/plugins/fetch-credentials)
-- [fetch-options](/packages/@ama-sdk/core/src/plugins/fetch-options)
-- [gateway-token](/packages/@ama-sdk/core/src/plugins/gateway-token)
-- [imperva](/packages/@ama-sdk/core/src/plugins/imperva)
-- [json-token](/packages/@ama-sdk/core/src/plugins/json-token)
-- [keepalive](/packages/@ama-sdk/core/src/plugins/keepalive)
-- [mock-intercept](/packages/@ama-sdk/core/src/plugins/mock-intercept)
-- [perf-metric](/packages/@ama-sdk/core/src/plugins/perf-metric)
-- [pii-tokenizer](/packages/@ama-sdk/core/src/plugins/pii-tokenizer)
-- [public-facts](/packages/@ama-sdk/core/src/plugins/public-facts)
-- [raw-response-info](/packages/@ama-sdk/core/src/plugins/raw-response-info)
-- [retry](/packages/@ama-sdk/core/src/plugins/retry)
-- [reviver](/packages/@ama-sdk/core/src/plugins/reviver)
-- [session-id](/packages/@ama-sdk/core/src/plugins/session-id)
-- [si-token](/packages/@ama-sdk/core/src/plugins/si-token)
-- [simple-api-key-authentication](/packages/@ama-sdk/core/src/plugins/simple-api-key-authentication)
-- [url-rewrite](/packages/@ama-sdk/core/src/plugins/url-rewrite)
-- [wait-for](/packages/@ama-sdk/core/src/plugins/wait-for)
-- [timeout](/packages/@ama-sdk/core/src/plugins/timeout)
+- [additional-params](src/plugins/additional-params)
```

Comment:
Is it correctly working on NPMjs.org package?

Reply:
replaced all relative links from the readme files